### PR TITLE
Streamlined readme and first draft of full installation script

### PR DESCRIPTION
**Pull request by maniospas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,156 +1,137 @@
...
 ```
 
-In case the script is not executable please run the command
+Kubeflow pipelines will be ready when all pods are in ready state.
+Afterwards,  access the Kubeflow Pipelines interface availabe at
+localhost:8080 by running:
 
 ```bash
-chmod +x kfp_install.sh
+kubectl port-forward --address 0.0.0.0 svc/ml-pipeline-ui 8010:80 -n kubeflow
```
Comment (georgiosn) [inline README.md:88]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,156 +1,137 @@
...
-When this script completes the Kubeflow pipelines deployment procedure started
-but it takes some time to be ready as many things need to be downloaded and
-configured during the deployment.
 
-You can check the status of the deployment with the following command:
+### Step 4: Toolkit startup
+
+Before proceeding please create an *.env* on the top level folder, if it does not already exists.
+Make sure that K3D with KFP installed is up and running and that the port
+forward to KFP is active. In this same top-level folder run the command
```
Comment (georgiosn) [inline README.md:99]:
```suggestion

Remember that port forward is activated with:
```bash
kubectl port-forward --address 0.0.0.0 svc/ml-pipeline-ui 8010:80 -n kubeflow
```

```

**Review by georgiosn — APPROVED** [review]
Thanks Manios!  I added some suggestions in the readme.

Also I don't currently have space or a clean test environment to run the installation script.

**Comment by maniospas** [comment]
Hi @georgiosn . There are some merge conflicts that you should resolve if possible. 
For versioning of new features, I suggest using the releases feature of github.
The installation script is still experimental (it's fine to have it be under development in my opinion if we mention this in the readme).

**Pull request merged**